x-pack/plugins/cloud/common/decode_cloud_id.ts

```diff
@@ -5,15 +5,15 @@
  * 2.0.
  */
 
+export interface DecodedCloudId {
+  host: string;
+  defaultPort: string;
+  elasticsearchUrl: string;
+  kibanaUrl: string;
+}
+
 // decodeCloudId decodes the c.id into c.esURL and c.kibURL
-export function decodeCloudId(cid: string):
-  | {
-      host: string;
-      defaultPort: string;
-      elasticsearchUrl: string;
-      kibanaUrl: string;
-    }
-  | undefined {
+export function decodeCloudId(cid: string): DecodedCloudId | undefined {
```

Comment:
Moved, pretty much without any modifications, to the `cloud` plugin. Tell me if you think anything within the method should be changed.

Reply:
Can't add a comment in that line. Should we use a different logger to replace all `console.debug` entries?

Reply:
Oh right, we have the client-side loggers now! will do.